geometry_msgs/PoseStamped finger
---
geometry_msgs/PoseStamped pruning_point

